diff -uNr lzmq-0.4.4/CMakeLists.txt lzmq-0.4.4_patched/CMakeLists.txt
--- lzmq-0.4.4/CMakeLists.txt	2019-08-11 17:02:56.110906294 +0200
+++ lzmq-0.4.4_patched/CMakeLists.txt	2019-08-11 17:05:33.211539369 +0200
@@ -26,6 +26,10 @@
 endif ( CMAKE_SYSTEM_NAME STREQUAL "Windows" )
 
 install_lua_module ( lzmq ${LZMQ_SRC} LINK ${LZMQ_LIBS} )
+# On mingw link all compiler libraries static.
+IF((CMAKE_SYSTEM_NAME STREQUAL "Windows") AND (${CMAKE_COMPILER_IS_GNUCC}))
+	SET_PROPERTY(TARGET lzmq PROPERTY LINK_FLAGS "-lgcc -lstdc++ -static-libgcc -static-libstdc++")
+ENDIF((CMAKE_SYSTEM_NAME STREQUAL "Windows") AND (${CMAKE_COMPILER_IS_GNUCC}))
 
 set_target_properties ( lzmq PROPERTIES COMPILE_DEFINITIONS "${LZMQ_DEFINE}" )
 set_target_properties ( lzmq PROPERTIES LINKER_LANGUAGE CXX)
